test/utils/test_normalization.py

```diff
@@ -67,7 +67,7 @@ def test_compute_x_laplacian_normalized_matrix():
     L = csr_matrix([[4.0, 0], [0.0, 4.0]])
     Lx = csr_matrix([[0.0, 1.0], [1.0, 0.0]])
     normalized_Lx = compute_x_laplacian_normalized_matrix(L, Lx)
-    expected_result = csr_matrix([[0.0, 0.25], [0.25, 0.0]])
+    expected_result = np.array([[0.0, 0.25], [0.25, 0.0]])
     assert np.allclose(normalized_Lx.toarray(), expected_result.toarray())
```

Comment:
The `toarray` call on `expected_result` is not necessary anymore after your changes.